bait-files/src/main/webapp/submission.js

```diff
@@ -132,7 +133,8 @@
 //       'Address Line 2', document.getElementById('address-input-div'));
 
 //   // City
-//   createTextInputElement('City', document.getElementById('address-input-div'));
+//   createTextInputElement('City',
```

Comment:
Why all the commented out stuff?

Reply:
I didn't want to delete this file in case Andrew wanted to reuse this code for what he plans to do.